audformat/core/database.py

```diff
@@ -514,11 +522,51 @@ def join_field(
                 elif value2:
                     self.__dict__[field] = value2
 
+        def update_scheme(
+                other: Database,
+                scheme: str,
+        ):
+            r"""Update schemes to contain the same labels"""
+            self_type = type(self.schemes[scheme].labels)
```

Comment:
I suggest to use `self.schemes.dtype` instead

Reply:
I changed it accordingly